Source/ObjectiveDropboxOfficial/Shared/Handwritten/OAuth/DBOAuth.h

```diff
@@ -140,6 +140,13 @@
 ///
 - (BOOL)clearStoredAccessTokens;
 
+///
+/// When NO, the Dropbox web login page will show a sign up link below the
```

Comment:
Still getting clarification on this, but we believe that it should be ok with a sign up link as the sign up page does not require getting a paid account that would otherwise need to be an in-app purchase.

I'll make that a little clearer in the document.